pkg/build/stage/from.go

```diff
@@ -42,20 +42,24 @@ func newFromStage(fromImageOrArtifactImageName, baseImageRepoIdOrNone, cacheVers
 type FromStage struct {
 	*BaseStage
 
-	fromImageOrArtifactImageName string
 	baseImageRepoIdOrNone        string
-	cacheVersion                 string
+	fromCacheVersion             string
+	fromImageOrArtifactImageName string
 }
 
 func (s *FromStage) HasPrevStage() bool {
 	return false
 }
 
 func (s *FromStage) GetDependencies(ctx context.Context, c Conveyor, cb container_backend.ContainerBackend, prevImage, prevBuiltImage *StageImage, buildContextArchive container_backend.BuildContextArchiver) (string, error) {
-	var args []string
+	args := make([]string, 0, len(s.configMounts)+4)
```

Comment:
Should we explicitly set the capacity in this case if 4 is a magic number that won’t adapt to code changes?

Reply:
Oops... I'd done the change thinking about code optimization but I forget to roll it back. I agree with you. I think we should use previous code version.